src/assets/scripts/client/info/AirportInfoController.js

```diff
@@ -0,0 +1,287 @@
+        /**
+         * Root DOM element
+         *
+         * @for AirportInfoController
+         * @property $element
+         * @type {jQuery|HTML element}
+         */
+        this.$element = $element;
+
+        /**
+         * Information div
+         *
+         * @for AirportInfoController
+         * @property $template
+         * @type {jQuery|HTML element}
+         */
+        this.$template = null;
+
+        /**
+         * @for AirportInfoController
+         * @property simClockController
+         */
+        this.simClockController = null;
+
+        /**
+         * Local instance of the event bus
+         *
+         * @for AirportInfoController
+         * @property _eventBus
+         * @type {EventBus}
+         */
+        this._eventBus = EventBus;
+
+        /**
+         * @for AirportInfoController
+         * @property wind
+         * @type {String}
+         */
+        this.wind = '';
+
+        /**
+         * @for AirportInfoController
+         * @property altimeter
+         * @type {Number}
+         */
+        this.altimeter = INVALID_NUMBER;
+
+        /**
+         * @for AirportInfoController
+         * @property elevation
+         * @type {String}
+         */
+        this.elevation = '';
+
+        /**
+         * @for AirportInfoController
+         * @property icao
+         * @type {String}
+         */
+        this.icao = '';
+
+        return this.init()
+                ._createChildren()
+                ._setupHandlers()
+                ._enable();
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method init
+     * @chainable
+     */
+    init() {
+        this.simClockController = new SimClockController();
+        this.$template = $(AIRPORT_INFO_TEMPLATE);
+        this.$clockView = this.$template.find(INFO_VIEW_SELECTORS.CLOCK_VALUE);
+        this.$windView = this.$template.find(INFO_VIEW_SELECTORS.WIND_VALUE);
+        this.$altimeterView = this.$template.find(INFO_VIEW_SELECTORS.ALTIMETER_VALUE);
+        this.$elevationView = this.$template.find(INFO_VIEW_SELECTORS.ELEVATION_VALUE);
+
+        return this;
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method _setupHandlers
+     * @chainable
+     * @private
+     */
+    _setupHandlers() {
+        this.onAirportChangeHandler = this.onAirportChange.bind(this);
+
+        return this;
+    }
+
+    /**
+     * Set initial element references
+     *
+     * Should be run once only on instantiation
+     *
+     * @for StripViewModel
+     * @method _createChildren
+     * @chainable
+     * @private
+     */
+    _createChildren() {
+        this.$element.append(this.$template);
+
+        return this;
+    }
+
+    /**
+     * Enable all event handlers
+     *
+     * @for AirportInfoController
+     * @method _enable
+     * @chainable
+     * @private
+     */
+    _enable() {
+        this._eventBus.on(EVENT.AIRPORT_CHANGE, this.onAirportChangeHandler);
+        this.onAirportChange();
```

Comment:
why do we call this in `._enable()`?  this would be a good spot for a comment that explains the _why_ here

Reply:
moved `this.onAirportChange();` into the constructor.